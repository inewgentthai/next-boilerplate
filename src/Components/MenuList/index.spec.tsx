// #region Global Imports
import * as React from "react";
// #endregion Global Imports

// #region Local Imports
import { render } from "@Test/utils";
import { MenuList } from "@Components";
// #endregion Local Imports

describe("MenuList", () => {
    it("should match snapshot", () => {
        const menus = {
            data: [
                {
                    id: 1,
                    name: {
                        en: 'en',
                        th: 'ไทย'
                    },
                    inventories: [
                        {
                            name: {
                                en: 'en',
                                th: 'ไทย'
                            },
                            price: 50
                        }
                    ]
                }
            ]
        }
        const { container } = render(<MenuList menus={ menus }>Test</MenuList>);

        expect(container).toMatchSnapshot();
    });
});
