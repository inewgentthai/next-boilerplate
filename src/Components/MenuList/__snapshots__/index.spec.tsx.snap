// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuList should match snapshot 1`] = `
<div>
  <section
    class="item-list"
  >
    <div
      class="container"
    >
      <div
        class="row no-gutters"
      >
        <div
          class="item"
        >
          <div
            class="paper"
          >
            <a
              data-backdrop="static"
              data-target="#itemModal"
              data-toggle="modal"
              href="#"
            >
              <div
                class="product-thumb"
              >
                <div
                  class="product-size"
                >
                  ไทย
                </div>
                <img
                  class="product-thumb"
                  srcset="undefined 2x, undefined 3x"
                />
              </div>
              <div
                class="product-info"
              >
                <div
                  class="product-type"
                >
                  Coffee
                </div>
                <div
                  class="product-name"
                >
                  ไทย
                   - 
                  ไทย
                </div>
                <div
                  class="product-price-add"
                >
                  <div
                    class="product-price"
                  >
                    ฿
                    50.00
                  </div>
                  <div
                    class="product-add icn-add-cart"
                  />
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
